client/src/main/scala/it/cwmp/client/view/AlertActor.scala

```diff
@@ -7,18 +7,27 @@ object AlertMessages {
   /**
     * Questo messagio serve per mostrare un messaggio all'utente di tipo informativo.
     *
-    * @param title contiene il titolo del messaggio
+    * @param title   contiene il titolo del messaggio
     * @param message contiene il corpo del messaggio da visualizzare
     */
   case class Info(title: String, message: String, onClose: Option[() => Unit] = None)
 
   /**
     * Questo messagio serve per mostrare un messaggio di errore all'utente.
     *
-    * @param title contiene il titolo del messaggio
+    * @param title   contiene il titolo del messaggio
     * @param message contiene il corpo del messaggio da visualizzare
     */
   case class Error(title: String, message: String, onClose: Option[() => Unit] = None)
+
+  /**
+    * Questo messagio serve per mostrare un messaggio di contenente il token della stanza privata appena creata all'utente.
+    *
+    * @param title   contiene il titolo del messaggio
+    * @param message contiene il corpo del messaggio da visualizzare
+    */
+  case class Token(title: String, message: String)
```

Comment:
In realtà il messagio di token è una cosa separata, che non fa parte degli alert generici che questa classe rappresenta... sposta questo messaggio tra i messaggi dell'attore dell'autenticazione